configure-data.tcl

```diff
@@ -243,23 +243,26 @@ proc postprocess {} {
 	}
 
 	if { $::HAVE_DARWIN } {
-		# ON Darwin there's a problem with linking against the Mac-provided OpenSSL.
-		# This must use brew-provided OpenSSL.
-		#
-		if { !$have_openssl || !$have_gnutls } {
-		
-			set er [catch {exec brew info openssl} res]
-			if { $er } {
-				error "You must have OpenSSL installed from 'brew' tool. The standard Mac version is inappropriate."
-			}
 
-			lappend ::cmakeopt "-DOPENSSL_INCLUDE_DIR=/usr/local/opt/openssl/include"
-			lappend ::cmakeopt "-DOPENSSL_LIBRARIES=/usr/local/opt/openssl/lib/libcrypto.a"
-		} elseif { $have_gnutls } {
+		if { $have_gnutls } {
 			set er [catch {exec brew info gnutls} res]
 			if { $er } {
 				error "Cannot find gnutls in brew"
 			}
```

Comment:
As you are fixing it here, is that ok that there are no extra cmake options added in case of gnutls, but they are in case of openssl? These explicit adding of options remain from the time when I haven't added automatic recognition of openssl libraries in cmake, so this was necessary. Can you please check if they are necessary maybe also in gnutls, or even on the contrary, none does?

Reply:
For `gnutls`, I added cmake option here, https://github.com/justinjoy/srt/blob/49643e99296c79f4d34ccc5348d4ed0fb142b482/configure-data.tcl#L214

And I assumed that you added openssl options here because only Mac has openssl problem, right?

Btw, shouldn't we use `cmake` directly? VLC complains `configure` is not compatible with normal options of autoconf's. Then, they started using cmake only. Please, refer to http://git.videolan.org/?p=vlc.git;a=blob;f=contrib/src/srt/rules.mak;h=f1e76ec7167c830cf0a18fac3c513cfd8ca364e6;hb=HEAD

Reply:
The `configure` script is rather predicted to be used "by finger"; it is only of autotools flavor, but otherwise it's only a wrapper for cmake. It does some more automatic checks than cmake and should be easier to use, but it's rather not predicted for automation. If you want to run it somehow automatically from within the VLC's build system, I'd recommend using cmake directly.

The initial idea for `configure` was because initially cmake was doing too many automation instead of relying on manual options, so I had a requirement to take it off; simultaneously I needed this automation, so I simply moved it to another script.